### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by testbird**
https://github.com/oetiker/znapzend/issues/312

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,8 +63,19 @@ Debian control files, guide on using them and experimental debian packages can b
 Configuration
 -------------
 
-Use the [znapzendzetup](doc/znapzendzetup.pod) program to define your backup settings. For remote backup, znapzend uses ssh.
-Make sure to configure password free login for ssh to the backup target host.
+For remote backups, znapzend uses ssh. So make sure to configure key based (passwordless) ssh login to the backup target host, so that znapzend is able to connect.
+
+Use `znapzendzetup`to create the desired backup properties within your ZFS pool.
```
Comment (oetiker) [inline README.md]: there is a space missing

Comment (oetiker) [inline README.md]: there is a space missing

**Comment by jimklimov** [comment]
Hello @testbird would you please update the PR so it is mergeable? Adding the bit on practical `znapzendzetup` usage right into the first doc one sees when coming to the project, would have saved me some torn hairs ;)

**Comment by testbird** [comment]
I'm currently busy and couldn't see right away why that conflict exists, it should just remove some fluff talk. Would you be able to adapt this?

**Comment by jimklimov** [comment]
Yes, I guess I can pull from your branch and post a new PR with reconciled file, so your commits stay authored by you ;)

**Pull request merged**